### ISSUE TITLE
Information about the pre-loaded wordsets (Dataloaders)

### ISSUE THREAD

**Issue by harshvr15** [dccuchile/wefe#24]
Hi Pablo,
Greetings!

1). I was trying to get results on multi-lingual fastText embeddings through the query using WEAT_wordsets() and RND_wordsets(). I am not sure whether I can use the same set of query for multi-lingual embeddings. However, I am getting some result for this.

WEAT_wordsets = load_weat()
RND_wordsets = fetch_eds()

glove_embedding = WordEmbeddings('de')
glove_keyed_vectors = glove_embedding.precomputed_word_embeddings
glove_100 = WordEmbeddingModel(glove_keyed_vectors, 'de')

result = weat.run_query(gender_3, glove_100, lost_vocabulary_threshold=0.3, warn_not_found_words=True, calculate_p_value=True, p_value_iterations=10000)
print(result)

**RESULT**:
{'query_name': 'Male terms and Female terms wrt Science and Arts', 'result': 0.50751936, 'weat': 0.50751936, 'effect_size': 0.9096829, 'p_value': 0.0034996500349965005}

But I am not sure whether it is correct?  Can you please guide whether these wordset queries can be used for multi-lingual embeddings and if not how can I proceed?

2). Can you please share any links where I can find information about the pre loaded wordsets **( load_weat() and fetch_eds() )**? 

For now, I just know:
fetch_eds(): Fetches the word sets used in the experiments of the work Word Embeddings *Quantify 100 Years Of Gender And Ethnic Stereotypes.
load_weat(): Loads the word sets used in the paper Semantics Derived Automatically From Language Corpora Contain Human-Like Biases.

I couldn't find any links on what data are they extracted from or what sets does it have and further information.

Furthermore,

For example: In this query:
gender_1 = Query(
    [RND_wordsets["male_terms"], RND_wordsets["female_terms"]],
    [WEAT_wordsets["career"], WEAT_wordsets["family"]],
    ["Male terms", "Female terms"],
    ["Career", "Family"],

(Correct me if I'm wrong) You are taking male_terms and female _terms as target set and career and family as attribute sets from these dataloaders. What are the other sets (like male_terms, adjectives_intelligence, female_terms etc.) which are there in this dataloaders? You have described a few in your queries, are there only those or are there few others which can be considered?

Hope to hear from you soon.

Thanks,
Harsh

**Comment by pbadillatorrealba** [comment]
Hi @harshvr15 

>  The difference is almost 9 times for the same model, what can be the possible reasons for these contradictions or how can we explain this, as I can see some other contradictions as well?

The main difference between WEAT and WEAT-ES is that WEAT-ES normalizes the results obtained using the average and standard deviation of the calculated statistics. This makes the WEAT-ES calculated scores independent of the number of words that each target and attribute possesses, which in practical terms allows obtaining much more robust results than using WEAT alone. This difference may explain the orders of magnitude difference between the different tests.

I have just updated the documentation with a better explanation of this: https://wefe.readthedocs.io/en/latest/about.html#weat

> Furthermore, I'm unable to interpret this for WEAT-ES : Since the ideal is also 0, we define ≤FWEAT-ES just as ≤FWEAT , can you please elaborate? Also, for RNSB.

On the other hand, the inequalities remain the same: the lower the absolute value of the score, the lower the bias detected.

In the case of RNSB, I think it is better to understand how the metric works :

Originally this metric is based on measuring bias through word sentiment. The main idea is that if there were no bias, all words should be equally negative. Therefore, its procedure is based on calculating how negative the words in the target sets are.

For this purpose, RNSB trains a classifier that assigns a probability to each word of belonging to the negative class (in the original work the classifier is trained using Bing Liu’s lexicon of positive and negative words). Then, it generates a probability distribution with the probabilities calculated in the previous step and compares them to the uniform distribution (case where all words have the same probability of being negative) using KL divergence. When the negative probability distribution is equal to the uniform one (i.e., there is no bias), the KL divergence is 0.

I also updated the documentation of this metric with the idea of making it a bit more understandable: 
https://wefe.readthedocs.io/en/latest/about.html#rnsb

Hope you find it useful 😊

Pablo.
